src/build/emitter.ts

```diff
@@ -826,6 +826,14 @@ export function emitWebIdl(
     entity: { comment?: string; deprecated?: boolean },
     print: (s: string) => void
   ) {
+    if (
+      entity.deprecated &&
+      entity.comment?.endsWith(`*/`) &&
```

Comment:
why not use `entity.comment?.startsWith("/*")` like line 838?

Reply:
This seemed to be more apt given the @deprecated tag gets inserted at the closing tag. In principle, given that a comment is valid and has an opening tag, it should have a closing tag a well, but there is no harm in checking.